.gitignore

```diff
@@ -3,9 +3,10 @@
 /.coverage
 /.pytest_cache
 /.tox
+/.vscode
```

Comment:
Some projects _do_ include some `.vscode` settings, so I actually prefer not to set this globally, and its for that reason that I personally do allow gitignore entries like this in my project. However, the link you gave above mentioned `.git/info/exclude`, which I wasn't aware of, so I'll do that to avoid having to add this line to your repository.